### PR TITLE
merge feat/challenge_03_loops-conditionals to merge

### PR DESCRIPTION
**Pull request by Daniel94y1**

### CODE REVIEW

File: challenge_03_loops-conditionals/index.js
Comment (tictools) [inline challenge_03_loops-conditionals/index.js:1]: ⚠️ FIX:remove file

File: challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_00.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    6. If it is neither "cat" nor "dog", print out "Must be an alien"
+    7. At the end of the program print out: "--- Animal check complete ---"
+    8. Change animal to different values to make sure it works in all cases
+*/
+
+console.log("--- Animal Checker Program ---");
+
+var animal = "frog";
+
+if (animal === "cat") {
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_00.js]: ⚠️ FIX: Change animal to different values to make sure it works in all cases

File: challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_01.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    The program will return the variable is even or odd.
+
+    Change the variable to test it works for different cases.
+    Print result.
+
+    HINT: modulus operator ( % ) is your friend
+*/
+var number = 8;
+
+if (number % 2 === 0) {
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_01.js]: ⚠️ FIX: Change animal to different values to make sure it works in all cases

File: challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+/*
+    1. Create a for loop that will print out all the even numbers between 10 and 40.
+    
+    2. Create a for loop that will print out all the odd numbers between 10 and 40.
+*/
+
+console.log("Even numbers between 10 and 40:");
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js]: You should take in account what `i` variable is storing. Is this variable referring to any index in this case?

⚠️ FIX: rename properly `i` variable

File: challenge_03_loops-conditionals/loops-conditionals/loops/exercise_02.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    Half way there!
+    Checkpoint! 60
+    Checkpoint! 70
+    Checkpoint! 80
+    Checkpoint! 90
+    You made it!
+    All, done!
+*/
+
+for (let i = 1; i <= 100; i++) {
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/loops/exercise_02.js]: You should take in account what `i` variable is storing. Is this variable referring to any index in this case?

⚠️ FIX: rename properly `i` variable

File: challenge_03_loops-conditionals/loops-conditionals/while/exercise_01.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+/*
+    1. Create a while loop that will print out all the even numbers between 10 and 40.
+
+    2. Create a while loop that will print out all the odd numbers between 10 and 40.
+*/
+let evenNumber = 10;
+console.log("Even numbers between 10 and 40:");
+while (evenNumber <= 40) {
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/while/exercise_01.js]: These loops should iterate over all numbers in between 10 and 40, printing only the odd/even ones, depending on its own logic.

⚠️ FIX: iterate over all numbers and print only the odd/even ones

File: challenge_03_loops-conditionals/loops-conditionals/while/exercise_02.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+    Checkpoint! 40
+    Half way there!
+    Checkpoint! 60
+    Checkpoint! 70
+    Checkpoint! 80
+    Checkpoint! 90
+    You made it!
+    All, done!
+*/
+let counter = 1;
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/while/exercise_02.js]: This is the output of your script...
```
Checkpoint! 10
Checkpoint! 20
Checkpoint! 30
Checkpoint! 40
Checkpoint! 50
Half way there!
Checkpoint! 60
Checkpoint! 70
Checkpoint! 80
Checkpoint! 90
Checkpoint! 100
You made it!
All, done!
```

...have you tested it before pushing? 🤔

⚠️ FIX: needs work.

**Review by tictools — CHANGES_REQUESTED** [review]

**Comment by Daniel94y1** [comment]
complete revision

File: challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js [hunk 7]
```diff
@@ -0,0 +1,26 @@
+/*
+    1. Create a for loop that will print out all the even numbers between 10 and 40.
+    
+    2. Create a for loop that will print out all the odd numbers between 10 and 40.
+*/
+
+function printEvenNumbers(start, end) {
+  console.log(`Even numbers between ${start} and ${end}:`);
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js:8]: 🚀

File: challenge_03_loops-conditionals/README.md
Comment (tictools) [inline challenge_03_loops-conditionals/README.md:1]: FIX: more file at root level. out of challenge_03 folder.

image

**Review by tictools — CHANGES_REQUESTED** [review]

File: challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_00.js
Comment (nuriaruiz97) [inline challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_00.js:1]: I think you can improve your code by simplifying the animalChecker function and eliminating redundant checks.

File: challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_01.js
Comment (nuriaruiz97) [inline challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_01.js:1]: nice!

File: challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_02.js
Comment (nuriaruiz97) [inline challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_02.js:1]: the problem in this exercise isn't in the logic conditions they give you, it's in the order of the conditions!

File: challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js
Comment (nuriaruiz97) [inline challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js:1]: nice!!

File: challenge_03_loops-conditionals/loops-conditionals/loops/exercise_01.js
Comment (nuriaruiz97) [inline challenge_03_loops-conditionals/loops-conditionals/loops/exercise_01.js:1]: check the  brackets

File: challenge_03_loops-conditionals/loops-conditionals/loops/exercise_02.js
Comment (nuriaruiz97) [inline challenge_03_loops-conditionals/loops-conditionals/loops/exercise_02.js:1]: check the brackets

File: challenge_03_loops-conditionals/loops-conditionals/while/exercise_00.js
Comment (nuriaruiz97) [inline challenge_03_loops-conditionals/loops-conditionals/while/exercise_00.js:1]: check the brackets

File: challenge_03_loops-conditionals/loops-conditionals/while/exercise_01.js
Comment (nuriaruiz97) [inline challenge_03_loops-conditionals/loops-conditionals/while/exercise_01.js:1]: Check the barckets, ensure consistent conditions for even and odd numbers in loops, and move the console.log statement outside the second while loop

**Review by tictools — APPROVED** [review]

**Review by nuriaruiz97 — APPROVED** [review]

**Pull request merged**